express/routes/players.js

```diff
@@ -22,12 +23,13 @@ router.get('/', validate(playersValidation.get), async (req, res) => {
 
 router.get('/:id', validate(playersValidation.get), async (req, res) => {
 	const player = await playersService.getById(req.params.id);
```

Comment:
Does it make sense to use `include` in this query to load both `player` / `playerTeamSeason` entities?